modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -232,6 +233,34 @@ public void testAddSystemObjectRelationship() throws Exception {
 		_testSystemObjectRelationshipManyToMany();
 	}
 
+	@Test
+	public void testMandatoryRelationship() throws PortalException {
```

Comment:
Can you include this case in the `testUpdateObjectRelationship`?

Reply:
Perfect.